Source/Mosa.DeviceSystem/Image.cs

```diff
@@ -6,6 +6,8 @@ public class Image
 	{
 		private int[] pixels;
 
+		public int[] Pixels => pixels;
```

Comment:
Should remain private.

Reply:
But then I can't make a fast DrawImage() function

Reply:
It's an implementation detail that should be hidden. Otherwise, it's leads to bad design.

Reply:
Can I make a GetPixels() function maybe or is it against design too?

Reply:
Use GetColor(x,y)

Reply:
But then DrawImage() is slow. It's actually the method that makes the biggest lost in performance (in Framebuffer at least)